aiidalab_widgets_base/computers.py

```diff
@@ -39,32 +40,24 @@ def extract_aiidacomputer_arguments(arguments):
     return valid_arguments(arguments, VALID_AIIDA_COMPUTER_SETUP_ARGUMETNS)
 
 class SshComputerSetup(ipw.VBox):
+    setup_counter = Int(0) # Traitlet to inform other widgets about changes
     def __init__(self, **kwargs):
         style = {"description_width":"200px"}
         computer_image = ipw.HTML('<img width="200px" src="./miscellaneous/images/computer.png">')
         self._inp_username = ipw.Text(description="SSH username:",
                                       layout=ipw.Layout(width="350px"),
+                                      contiunous_update=False,
```

Comment:
continuous_update

Reply:
ah, this I actually wanted to delete. Sorry, leftover from my tests

Reply:
fixed